### PR TITLE
Fix percentage in flexing parent

### PR DESCRIPTION
**Pull request by woehrl01**
If we don't measure exactly, percentage values aren't exactly either. Fix for #414.

### CODE REVIEW

File: yoga/Yoga.c [hunk 1]
```diff
@@ -2339,7 +2339,12 @@ static void YGNodelayoutImpl(const YGNodeRef node,
                                           availableInnerCrossDim) +
                            marginCross;
           childCrossMeasureMode =
-              YGFloatIsUndefined(childCrossSize) ? YGMeasureModeUndefined : YGMeasureModeExactly;
+              YGFloatIsUndefined(childCrossSize) ||
+                      (currentRelativeChild->resolvedDimensions[dim[crossAxis]]->unit ==
```
Comment (astreet) [inline yoga/Yoga.c]: Could we extract this condition into a named bool? Or add comments on the two conditions we're trying to check here?

Comment (woehrl01) [inline yoga/Yoga.c]: Sure, will do

**Comment by emilsjolander** [comment]
@woehrl01 I think you accidentally added a bunch of `flex-basis` tests into this PR.

**Comment by woehrl01** [comment]
@emilsjolander ups, sorry! here you are! 😄

**Pull request closed without merging**